### ISSUE TITLE
[Feature Request] Adding a Button to Copy Image to Clipboard

### ISSUE THREAD

**Issue by RoyiAvital** [plotly/plotly.js#6888]
When using Plotly in Jupyter (Either with Python or Julia) I appreciate the interactivity of Plotly.  
It is a great experience compared to Matplotlib.

Yet I miss the ability to copy the image into clipboard in a click.  
I'd like to have a button, just like the `Download PNG` button which instead of downloading the file will copy the bitmap into the clipboard.

It seems users are looking for such feature, though for now they try creating a bypass:

 - https://github.com/plotly/plotly.js/issues/6060.
 - https://github.com/plotly/plotly.py/issues/2114.

Any chance to have this feature added built in?

**Comment by alexcjohnson** [comment]
Good idea, I could even see this becoming a default addition to the modebar.

We have a lot of work in our queue already so would probably take us a while to get to this, but we'd gladly review a PR, should be a relatively straightforward addition.

**Comment by rmorshea** [comment]
I think [browser compatibility](https://developer.mozilla.org/en-US/docs/Web/API/ClipboardItem#browser_compatibility) around the `clipboard` API has only recently improved. I suspect that's the main reason this hasn't been implemented before. Here's [an example](https://developer.mozilla.org/en-US/docs/Web/API/ClipboardItem#examples) showing how to do it for anyone who's interested in making a contribution.

**Comment by DanyMolinaN** [comment]
ntegrated the latest changes from the remote master branch into the local repository.
Manually resolved merge conflicts in the following files:

.github/ISSUE_TEMPLATE.md

.gitignore

CONTRIBUTING.md

LICENSE

README.md

During the merge, preserved and integrated the local implementation of a custom modebar button example within the CustomBundle build, ensuring compatibility with the current Plotly.js codebase.
Also synced documentation and configuration files with upstream while maintaining local customizations.
